Content.Server/Emp/EmpSystem.cs

```diff
@@ -101,7 +106,7 @@ private void OnCameraSetActive(EntityUid uid, EmpDisabledComponent component, re
 }
 
 [ByRefEvent]
-public record struct EmpPulseEvent(float EnergyConsumption, bool Affected, bool Disabled);
+public record struct EmpPulseEvent(float EnergyConsumption, bool Affected, bool Disabled, TimeSpan Duration);
```

Comment:
If these are supposed to have defaults then please set defaults.

Reply:
I'm not sure if they're meant to have defaults? I didn't write the system.
If you'd like me to add defaults, I can, but it would probably be more logical to add it to the function rather than the event for clarity during use.